packages/grid/test/keyboard-navigation.test.js

```diff
@@ -1889,6 +1890,31 @@ describe('keyboard navigation', () => {
 
       grid.removeEventListener('cell-focus', spy);
     });
+
+    // Chrome uses a workaround to dispatch cell-focus in mouse up,
+    // should dispatch event on mouse up events on cell itself
+    (isChrome ? it : it.skip)('should dispatch cell-focus on mouse up inside of cell', () => {
```

Comment:
This test seems to verify that cell-focus is dispatched when clicking directly on `cell._content`, not inside so the test name may need to be adjusted accordingly.

In addition to this test, it might be useful to have also a test verifying the same but for a mouse up event fired inside of the cell.

Reply:
Updated the description and comments. Testing for the cell itself (`td` element) does not work, as the event listener is on the document, and the cell is in the grid's shadow DOM, so the event will have the grid as target. We could add a second mouse up listener on the cell itself, however from my testing the content element covers the whole cell, so I think that's not necessary.

Reply:
Correction, the cell content element does not cover the whole cell, but it seems the workaround is only required when clicking on the content. When clicking on the cell itself, the focus event triggers normally, on mouse down.

Reply:
Yeah, got it, but what about the case where the user clicks somewhere inside the cell content, not directly on the cell content element? What I'm thinking of is whether there is a necessity for an additional test that will verify that the `contains` method is used and not just `event.target === cellContent`.

Reply:
Added a test for clicking on a child of the cell content, and moved all Chrome related tests into a new suite to remove the duplicate comments.